### PR TITLE
Style/gdh 746 margins components wp spec

### PR DESCRIPTION
**Pull request by robbieatacato**
### Purpose:

- re-add margins components, excluding: accordion, anchor navigation, card authentication, cta image content

closes #1132

### CODE REVIEW

File: components/Typography/src/heading.scss [hunk 1]
```diff
@@ -81,3 +81,9 @@
     --utrecht-heading-5-margin-block-start: var(--denhaag-space-block-2xl);
   }
 }
+
+/* stylelint-disable */
+.page-container [class^="utrecht-heading"] + *:not(.denhaag-button),
```
Comment (paulAcato) [inline components/Typography/src/heading.scss:86]: This is still to global,  i applied this style locally and still blocks like the authentication block breaks.

File: components/Typography/src/paragraph.scss [hunk 2]
```diff
@@ -40,3 +40,18 @@
 .denhaag-list__wrapper + .utrecht-paragraph--lead {
   --utrecht-paragraph-margin-block-start: var(--utrecht-paragraph-lead-list-margin-block-start, inherit);
 }
+
+/* stylelint-disable */
+@media (min-width: 768px) {
+  .page-container
+    *:not(.utrecht-paragraph):not([class^="utrecht-heading"]):not([class*=" utrecht-heading"])
```
Comment (paulAcato) [inline components/Typography/src/paragraph.scss:47]: Idem

File: components/Typography/src/paragraph.scss [hunk 3]
```diff
@@ -40,3 +40,18 @@
...
+    + .utrecht-paragraph {
+    margin-block-start: var(--denhaag-space-block-2xl);
+  }
+
+  .page-container
+    .utrecht-paragraph
+    + *:not(.utrecht-paragraph):not([class^="utrecht-heading"]):not([class*=" utrecht-heading"]) {
```
Comment (paulAcato) [inline components/Typography/src/paragraph.scss:54]: Idem

File: components/Typography/src/heading.scss (hunk 1 above)
Comment (paulAcato) [inline components/Typography/src/heading.scss:86]: `page-container` is no valid NLDS classname

**Comment by robbieatacato** [comment]
Ik heb de .page-container vervangen met de component classes die uitgesloten moeten worden:

- accordion, 
- anchor navigation, 
- card authentication, 
- cta image content

**Pull request closed without merging**